plaso/parsers/winevtx.py

```diff
@@ -292,6 +297,50 @@ def _ParseRecords(self, parser_mediator, evtx_file):
             'unable to parse recovered event record: {0:d} with error: '
             '{1!s}').format(record_index, exception))
 
+  def _ElementTreeToDict(
```

Comment:
Please follow the plaso style guide https://github.com/log2timeline/l2tdocs/blob/master/process/Style-guide.md